css/portal-dashboard/feedback/rubric-table.less

```diff
@@ -0,0 +1,167 @@
+@import "../variables";
+
+.rubricContainer {
+  border: solid 1.5px @cc-charcoal-light2;
+  border-radius: 8px;
+  background-color: white;
+  margin: 10px 0;
+  width: 100%;
+  font-size: 14px;
+
+  .columnHeaders {
+    height: 45px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    text-align: center;
+
+    .rubricDescriptionHeader {
+      display: flex;
+      align-items: center;
+      flex: auto;
+
+      .scoringGuideArea {
+        width: 100px;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+
+        .launchButton {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          cursor: pointer;
+          height: 30px;
+          width: 30px;
+          margin: 0 11px;
+          background-color: @feedback-green-light4B;
+          border: solid 1.5px @cc-charcoal-light1;
+          border-radius: 8px 0;
+          box-sizing: border-box;
+
+          &:hover {
+            background-color: @feedback-green-light3B;
+          }
+          &:active {
+            color: white;
+            background-color: @feedback-green;
+
+            .icon {
+              fill: white;
+            }
+          }
+
+          .icon {
+            width: 24px;
+            height: 24px;
+            pointer-events: none;
+            fill: @feedback-green;
+          }
+        }
+      }
+
+      .rubricDescriptionTitle {
+        justify-content: center;
+        width: 75%
```

Comment:
Percents like this can result in odd effects at big and small widths.  That said, this looked good in practice.